content/en/account_management/audit_logs.md

```diff
@@ -0,0 +1,97 @@
+
+## Configuration
+
+### Event types
+
+Event types are a collection of audit events. For example, the Authentication event type contains all logs related to authentication and the Dashboards event type contains all the logs related to interacting with the dashboards product. To enable an event type, navigate to the *Audit Logs Settings* section of your [Organization Settings][2] and toggle on event types that are relevant to you.
+
+{{< img src="account_management/audit_logs/event-types.png" alt="Audit Logs event types setup in Datadog" style="width:70%;">}}
+
+### Archiving
+
+Archiving is an optional feature for Audit Logs. You can use archiving to write to Amazon S3, Google Cloud Storage, or Azure Storage and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written within 15 minutes of generation.
+
+To enable archiving for Audit Logs, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Scroll down to Archiving and click the Store logs toggle to enable.
+
+### Retention
+
+Retaining logs is an optional feature for Audit Logs. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Scroll down to Retention and click the Retain logs toggle to enable.
+
+The default retention period for an audit log is seven days. You can set a retention period between three and 90 days.
+
+{{< img src="account_management/audit_logs/retention.png" alt="Audit Logs Retention setup in Datadog" style="width:80%;">}}
+
+Note: Audit Logs are priced as retained logs, and there is no cost for ingestion or archiving. See the [Log Management pricing page][3] for more information.
+
+## Explore Audit Logs
+
+To explore an audit log, navigate to the [Audit Logs][1] section, also accessible from your [Organizational Settings][2] in Datadog.
+
+{{< img src="account_management/audit_logs/explore-audit-logs.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+Audit Logs have the same functionality as logs within the Datadog [Logs Explorer][4]:
+
+- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Info`) or `Method` (`POST`, `GET`, `DELETE`).
+- For compliance audits, filter by `event_name` and select `authentication only` to see compliance-related events.
+- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.
+
+{{< img src="account_management/audit_logs/attributes.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+## Create a monitor
+
+To create a monitor on a type of audit log or by specific log attributes, see the [Audit Logs Monitor documentation][5].
```

Comment:
In case we want to add examples - Set a monitor that triggers when user XYZ logs in, or set a monitor for anytime a dashboard is deleted